DOCUMENTATION.md

```diff
@@ -842,6 +862,28 @@ The Content API is defined on your set of rules in the configuration file, the o
 - `DELETE /api/users/ID`
   - Access limited to own user or users with permission `user.admin` and `user.delete`. For the current user is also required to send a `password` query parameter.
 
+### Groups 
+- `GEt /api/groups/[GROUP_TYPE]`
+    - Access limited to users with permission `group.TYPE.read` or `group.TYPE.admin`
+- `GET /api/groups/[GROUP_TYPE]/[GROUP_SLUG]` | `GET /api/content/[GROUP_TYPE]/[GROUP_ID]?field=id`
+  - Access limited to own user or users with permission `group.TYPE.read` or `group.TYPE.admin`
+- `GET /api/groups/[GROUP_TYPE]/[GROUP_SLUG]/users`
+  - Access limited to own user or users with permission `group.TYPE.read` or `group.TYPE.admin`
```

Comment:
the permissions are not correct it would be `group.TYPE.users.read` ,  `group.TYPE.users.admin`  and ,  `group.TYPE.admin`